test/check-storage-reclaim

```diff
@@ -369,6 +412,13 @@ class TestStorageReclaimExistingSystemWindows(VirtInstallMachineCase):
 
     @disk_images([("", 20)])
     def testBasic(self):
+        """
+        Description:
+            Test reclaiming space from an existing Windows system by deleting partitions
+
+        Expected results:
+            - The review screen shows the affected system when partitions from an existing system are deleted
```

Comment:
I think this test (compared to the one above (`testDeletePartition`) is missing the review screen check.

Reply:
Did not add the check, because I wanted to keep this commit clean from code changes. Adjusted the docstring, and we can follow up with improvements of the code itself.